### ISSUE TITLE
BUG: randomly attack command does not work after starting new game

### ISSUE THREAD

**Issue by ballotz** [QW-Group/ezquake-source#875]
Could be related to #283 

replicated building from master branch fa5d11e14091a8fb97c46090f4e1d1030f049a3b

Windows 10, built with VS2019, platform toolset v142

1 - start new game from main menu
2 - press fire
3 - if fire works, go to step 1
4 - pressing the fire button (both mouse or keyboard) does not make weapon to fire, other controls seems to work properly

I can replicate it after a few tries, (usually from 3 to 5) with a debug build in windowed mode.

It is a bit difficult to debug because the bug goes away when the game window looses and gets focus again.
To debug it I put a breakpoint in keys.c:2111, then repeat the steps above firing with the mouse, so the breakpoint does not hit.
Then when the bug shows, I hit CTRL to fire so the breakpoint is hit while the bug is happening.
Following the attack command seems that the problem is on the server.
sv_user.c: 4561
for some reason, randomly Info_Get(&cl->_userinfoshort_ctx_, "chat"); does not return an empty string so the attack command is masked away.

**Comment by ballotz** [comment]
I noticed that he code mentioned above that I suppose is causing the bug, is under the CHAT_ICON_EXPERIMENTAL #ifdef.
That macro is defined in server.h and seems coming from an old 2008 commit 358380cd51ff29fb35a9abbf790dbe9ce2f602a9
The name experimental does not sounds promising, but maybe it is a currently used feature, idk
Wonder if it can be just removed

**Comment by dsvensson** [comment]
Is this something that resurfaced with https://github.com/QW-Group/ezquake-source/commit/fbcf8a89f57da1902021cfb8a43dd109e9ccfdb7 ?

**Comment by ballotz** [comment]
Yes, I checked on 580b371e0de06de918c034589f6df63474b4a44e and the issue occurred at third try.
Reading the comments under #ifdef CHAT_ICON_EXPERIMENTAL seems that that code is preventig user from firing while in console. Maybe some variable is randomly initialized and after starting a new game the server thinks that the console is opended?
